src/VSMSBuildExtensions/VSMSBuildExtensions.proj

```diff
@@ -56,6 +56,7 @@
       <VSMSBuildExtensionsContent Include="$(ArtifactsBinDir)Microsoft.DotNet.MSBuildSdkResolver\$(Configuration)\net472\arm64\hostfxr.dll" DeploymentSubpath="MSBuildSdkResolver/arm64/" />
       <VSMSBuildExtensionsContent Include="$(ArtifactsBinDir)Microsoft.DotNet.MSBuildSdkResolver\$(Configuration)\net472\**\Microsoft.Deployment.DotNet.Releases*.dll" DeploymentSubpath="MSBuildSdkResolver/" />
       <VSMSBuildExtensionsContent Include="$(ArtifactsBinDir)Microsoft.DotNet.MSBuildSdkResolver\$(Configuration)\net472\**\Microsoft.DotNet.MSBuildSdkResolver*.dll" DeploymentSubpath="MSBuildSdkResolver/" />
+      <VSMSBuildExtensionsContent Include="$(ArtifactsBinDir)Microsoft.DotNet.MSBuildSdkResolver\$(Configuration)\net472\SdkResolvers\**\Microsoft.DotNet.MSBuildSdkResolver.xml" DeploymentSubpath="MSBuildSdkResolver/" />
```

Comment:
@joeloff I think this is the right place to add the generated resolver XML to ensure it gets added to the VSIX, but I don't know how to create/inspect the VSIX to verify. Any guidance I can follow here?

Reply:
The VSIX is just a zip, but it doesn't get produced in the SDK. We only generate the source (.swr file) and VS generates the actual zip, but if you do `build -pack`, you should end up with a file under `artifacts\packages\Debug\NonShipping`.

Reply:
Thanks, that's very helpful! I cracked open the SWR and found:

```
folder "InstallDir:\MSBuild\Current\Bin\SdkResolvers\Microsoft.DotNet.MSBuildSdkResolver\Microsoft.DotNet.MSBuildSdkResolver\"
  file source="$(PkgVS_Redist_Common_Net_Core_SDK_MSBuildExtensions)\MSBuildSdkResolver\Microsoft.DotNet.MSBuildSdkResolver\Microsoft.DotNet.MSBuildSdkResolver.xml"
```

That to me suggests that I have the layout incorrect because I believe the xml needs to be either adjacent to the dll, or in the `SdkResolvers\<ResolverName>\<ResolverName>.xml` layout pattern. So I'll need to go fix that.

That's validated by digging into the nupkg as well:
![image](https://github.com/user-attachments/assets/3e879743-2633-4927-8873-252a97c5e909)

Here I can see that the directory structure is incorrect.